conf/test.config

```diff
@@ -70,21 +70,12 @@ process {
         }
     }
 
-    if (params.joint_mutect2) {
-        withName: 'MUTECT2_PAIRED' {
-            ext.args         = { params.ignore_soft_clipped_bases ?
-                                    "--dont-use-soft-clipped-bases true --f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample ${meta.normal_id}" :
-                                    "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample ${meta.normal_id}" }
-        }
-    }
-    else {
-        withName: 'MUTECT2_PAIRED'{
-            //sample name from when the test data was generated
-            ext.args = { "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample normal " }
-        }
+    withName: '.*:MUTECT2_PAIRED'{
+        //sample name from when the test data was generated
+        ext.args = { "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz --normal-sample normal" }
     }
 
-    withName: 'FILTERVARIANTTRANCHES'{
+    withName: '.*:FILTERVARIANTTRANCHES'{
```

Comment:
I see you made a similar change elsewhere in this PR. What is the advantage of adding `.*:`?

Reply:
so that we overwrite the default value in the actual configs

Reply:
Thanks for enlighting me on this. I would have expected `withName: 'FILTERVARIANTTRANCHES'` to also overwrite the values from the "actual" configs.

Reply:
`.*` has priority over more specific selectors